types/file-saver/tslint.json

```diff
@@ -1,6 +1,7 @@
 {
     "extends": "dtslint/dt.json",
     "rules": {
-        "export-just-namespace": false
+        "export-just-namespace": false,
+        "unified-signatures": false
```

Comment:
Maybe it will be better if you disable this rule on the line where it interferes?

```ts
// tslint:disable-next-line:unified-signatures
```